packages/perseus/src/types.ts

```diff
@@ -228,6 +228,8 @@ export type APIOptions = Readonly<{
     // This is useful for when we need to know how a widget has shuffled its
     // the available choices.
     onWidgetStartProps?: (widgets: PerseusWidgetMap) => PerseusWidgetMap;
+    // Feature flags that can be passed from consuming application
+    flags?: Record<string, boolean | Record<string, boolean>>;
```

Comment:
I believe the below would also work, but I don't have super strong feelings one way or the other:

```ts
this.props.apiOptions?.flags?.[`mafs-${this.props.graph.type}-graph`] &&
mafsGraphTypes.includes(this.props.graph.type)
```

Reply:
I like the use of optional chaining to simplify the check.

Reply:
**thought**: Can we be more explicit with what the flags can contain here?  I think at "compile time" we should know what flags Perseus supports.  That'd help integration too as we'd have invalid types in the host app if they try to provide flag keys that we don't support. 

This will also help in the Perseus code so that we don't try to read flags that will never be populated. 

So something like this: 

```js 
 // Feature flags that can be passed from consuming application
     flags?: {
         mafs?: {
             segment?: true // or boolean
         }
     }
```